includes/Checker/Checks/Abstract_PHP_CodeSniffer_Check.php

```diff
@@ -122,16 +127,25 @@ final public function run( Check_Result $result ) {
 			}
 
 			foreach ( $file_results['messages'] as $file_message ) {
-				$result->add_message(
-					strtoupper( $file_message['type'] ) === 'ERROR',
-					$file_message['message'],
-					array(
-						'code'   => $file_message['source'],
-						'file'   => $file_name,
-						'line'   => $file_message['line'],
-						'column' => $file_message['column'],
-					)
-				);
+				if ( strtoupper( $file_message['type'] ) === 'ERROR' ) {
+					$this->add_result_error_for_file(
+						$result,
+						$file_message['message'],
+						$file_message['source'],
+						$file_name,
+						$file_message['line'],
+						$file_message['column']
+					);
+				} else {
+					$this->add_result_warning_for_file(
+						$result,
+						$file_message['message'],
+						$file_message['source'],
+						$file_name,
+						$file_message['line'],
+						$file_message['column']
+					);
+				}
```

Comment:
Code like this would be much simpler if we just had 1 method with a `$severity` parameter, instead of two separate methods that are nearly identical.

Reply:
Thanks for the feedback. Addressed in https://github.com/WordPress/plugin-check/pull/295/commits/95e863e339a7c7d757aa2d2076b11cadcc0e21b7